src/prettytable/prettytable.py

```diff
@@ -1522,9 +1518,15 @@ def copy(self):
 
     def _format_value(self, field, value):
         if isinstance(value, int) and field in self._int_format:
-            return ("%%%sd" % self._int_format[field]) % value
+            int_format = self._int_format.get(field, "")
+            if not int_format[-1:].isalpha():
+                int_format += "d"
```

Comment:
Any particular reason to use f-strings in the validate functions and concatenation here?

Reply:
I wasn't certain that references to str variables would change in a function, but these are immutable so it's fine to use `+=` for both instances.